extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -0,0 +1,143 @@
+[TBD - point to relevant filtering algorithm]  
+If a compressed texture format is used then pre-filtered mip-levels must be included.  
+
+## Declaring an environment light
+
+The KHR_lights_environment extension defines an array of image-based lights at the root of the glTF and then each scene can reference one.  
+Each environment light definition consists of a single cubemap that describes the specular radiance of the scene, the l=2 spherical harmonics coefficients for diffuse irradiance and intensity values.  
+The cubemap is defined by texture references to a KTX2 file containing a cubemap, these files can contain compressed textures using KHR_texture_basisu  
+or use a float texture format as defined by KHR_texture_float_bc6h or KHR_texture_float_rgb95e.   
+
+
+```json
+"extensions": {
+    "KHR_lights_environment" : {
+        "lights": [
+            {
+                "intensity": 1.0,
+                "irradianceCoefficients": [...3 x 9 array of floats...],
+                "specularCubemaps": [... Reference to KTX2 cubemaps...],
+            }
+        ]
+    }
+}
+```
+
+## Specular Radiance Cubemaps
+
+The cubemap used for specular radiance is defined as a cubemap containing separate images for each cube face. 
+The mip levels shall evenly map to roughness values from 0 to 1 in the PBR material and should be generated with a principled multi-scatter GGX normal distribution. The data in the maps represents illuminance in candela per square meter (nit).
+
+If mip-levels are not included, then the entire mip chain of images should not be generated.  
+Instead, the lowest-resolution mip should have sufficient size to represent the maximally-blurred radiance map (say, 16x16) corresponding to roughness=1. The texture references defines the largest dimension of mip 0 and, taken together with the number of defined mips, should give the loading runtime the information it needs to generate the remainder of the mip chain and sample the appropriate mip level in the shader.
+
+Cube faces are defined in the KTX2 format specification, users of this extension must follow the KTX2 cubemap specification.  
+
+
+## Irradiance Coefficients
+
+This extension uses spherical harmonic coefficients to define irradiance used for diffuse lighting.  
+Coefficients are calculated for the first 3 SH bands (l=2) and take the form of a 9x3 array.  
+[Realtime Image Based Lighting using Spherical Harmonics](https://metashapes.com/blog/realtime-image-based-lighting-using-spherical-harmonics/)
+[An Efficient Representation for Irradiance Environment Maps](http://graphics.stanford.edu/papers/envmap/)
+
+### Using the environment light
+
+The environment light is utilized by a scene.  
+
+```json
+"scenes": [
+    {
+        "nodes": [
+            0
+        ],
+        "extensions": {
+            "KHR_lights_environment": {
+                "environment": 0
+            }
+        }
+    }
+  ]
+```
+
+## Adding Light Instances to Scenes
```

Comment:
The following section is duplicated with the above "Using the Environment Light" section.

Reply:
Removed this section